manifest.php

```diff
@@ -48,7 +48,7 @@
     'label' => 'Proctoring',
     'description' => 'Proctoring for deliveries',
     'license' => 'GPL-2.0',
-    'version' => '17.2.2',
+    'version' => '17.2.3',
```

Comment:
Deprecating classes and removing public methods doesn't seem like a PATCH version change.

Reply:
I didn't change any behaviour, business logic and class did not lost any dependencies. It was `just` refactoring so it's just a patch that is contained in [132b0141391e46b30869e9af051678f5f82d6202](https://github.com/oat-sa/extension-tao-proctoring/pull/730/commits/132b0141391e46b30869e9af051678f5f82d6202)